spec/Section 2 -- Language.md

```diff
@@ -1094,3 +1095,24 @@ and operations.
 
 As future versions of GraphQL adopt new configurable execution capabilities,
 they may be exposed via directives.
+
+
+## Document Directives
```

Comment:
Don't think they should be a special case of directives.
They should be covered by: https://facebook.github.io/graphql/June2018/#sec-Language.Directives

Reply:
Intent is place document level directives aside from others, there apply different rules:
- Document directives may be repeated, others not.
- Document level directives are top level **AST Document** `DocumentNode.definitions` nodes, other directives are listed in `XxxNode.directives` — this may need discussion

Reply:
> Document directives may be repeated, others not.

#472 add support of reputable directives.

> Document level directives are top level AST Document DocumentNode.definitions nodes, other directives are listed in XxxNode.directives — this may need discussion

I think `DocumentNode.directives` makes more sense.